### ISSUE TITLE
Sign in with Google temporarily disabled for this app

### ISSUE THREAD

**Issue by vrobert78** [cloudyr/bigQueryR#63]
Hi,

we have a datascientist who use bigQueryR. Until now it was working well.
But today he had an error: 

"
Sign in with Google temporarily disabled for this app.
This app has not been verified yet by google in order to use google sign in
"

It seems that you must to go through the verification process: https://support.google.com/cloud/answer/7454865?hl=en

Do you agree ? Is there something we can do by ourselves ?

**Comment by MarkEdmondson1234** [comment]
Yes Google has changed OAuth2 criteria - the default client.id that comes with the library was verified but seems to have lapsed again.  To get around it use your own client.id and secret which will let the admin owner through for internal users.  That is best achieved using `googleAuthR::gar_set_client()` before authentication is attempted - see googleAuthR website for details.

**Comment by vrobert78** [comment]
Thanks for such a fast reponse !

I understand that we have two workaround: 

1- to wait until the default client.ip will be re-approved. Do you have an idea of the ETA ?
2- to use our own client.ip & secret

Could you confirmed ?

**Comment by MarkEdmondson1234** [comment]
Yep thats it.  I'll try to re-auth this week but its a bit of a mystery why its been de-authed so long term I would recommend use your own client.id
